docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -279,7 +279,7 @@ Notice that `get()` and `set()` methods are used for retrieving and setting on-c
 
 - Similarly, using `set()` changes the transaction to indicate that changes to this particular on-chain state are updated only when the transaction is received by the Mina network if it contains a valid authorization (usually, a valid authorization is a proof).
 
-The logic also uses the `.mul()` method for multiplication of the values stored in `Field` types. You can view all available methods in the [o1js Reference](/zkapps/o1js-reference) documentation. 
+The logic also uses the `.mul()` method for multiplication of the values stored in `Field` types. You can view all available methods in the [Field](/zkapps/o1js-reference/classes/Field) in the o1js Reference documentation. 
```

Comment:
```suggestion
The logic also uses the `.mul()` method for multiplication of the values stored in `Field` types. You can view all available [methods](/zkapps/o1js-reference/classes/Field#methods) in the o1js Reference documentation. 
```

Reply:
wdyt about linking directly to methods and updating the link text to match, did you know we have a docs style guide?  https://github.com/o1-labs/docs2/wiki/Docs-Style-Guide

Use short [descriptive link text](https://developers.google.com/style/link-text) that helps provide context for the material that you're linking to

Reply:
@barriebyron nice update, thanks